webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -3233,6 +3234,7 @@ object RudderConfigInit {
 
       new CheckRudderGlobalParameter(roLDAPParameterRepository, woLDAPParameterRepository, uuidGen),
       new CheckInitXmlExport(itemArchiveManagerImpl, personIdentServiceImpl, uuidGen),
+      new MigrateEventLogEnforceSchema(doobie),
```

Comment:
this migration check needs to be earlier in the stack, I would say just after checking that connection are up (ie after `CheckConnections`). You want migration to happen as early as possible so that your business code doesn't have to worry that some invariant does not hold (not relevant in that case, but a general good practice)